### PR TITLE
Publisher Mainnet Release

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: packages/shared/dist/components/JSONtoUl/index.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ function _slicedToArray(arr, i) { return _arrayWithHoles(arr) || _iterableToArra
 
 function _nonIterableRest() { throw new TypeError("Invalid attempt to destructure non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method."); }
```
Comment (raamb) [inline packages/shared/dist/components/JSONtoUl/index.js:21]: Is this expected?

File: packages/shared/dist/components/SNETForgotPasswordConfirm/index.js [hunk 2]
```diff
@@ -37,7 +39,7 @@ function _slicedToArray(arr, i) { return _arrayWithHoles(arr) || _iterableToArra
 
 function _nonIterableRest() { throw new TypeError("Invalid attempt to destructure non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method."); }
 
-function _unsupportedIterableToArray(o, minLen) { if (!o) return; if (typeof o === "string") return _arrayLikeToArray(o, minLen); var n = Object.prototype.toString.call(o).slice(8, -1); if (n === "Object" && o.constructor) n = o.constructor.name; if (n === "Map" || n === "Set") return Array.from(n); if (n === "Arguments" || /^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(n)) return _arrayLikeToArray(o, minLen); }
+function _unsupportedIterableToArray(o, minLen) { if (!o) return; if (typeof o === "string") return _arrayLikeToArray(o, minLen); var n = Object.prototype.toString.call(o).slice(8, -1); if (n === "Object" && o.constructor) n = o.constructor.name; if (n === "Map" || n === "Set") return Array.from(o); if (n === "Arguments" || /^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(n)) return _arrayLikeToArray(o, minLen); }
```
Comment (raamb) [inline packages/shared/dist/components/SNETForgotPasswordConfirm/index.js:42]: Is this expected?

File: packages/shared/dist/components/SNETHeader/MobileHeader/index.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ function _slicedToArray(arr, i) { return _arrayWithHoles(arr) || _iterableToArra
 
 function _nonIterableRest() { throw new TypeError("Invalid attempt to destructure non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method."); }
```
Comment (raamb) [inline packages/shared/dist/components/SNETHeader/MobileHeader/index.js:31]: Is this expected?

File: packages/shared/src/components/SNETForgotPassword/index.js [hunk 4]
```diff
@@ -35,7 +37,7 @@ const SNETForgotPassword = ({ title, email, forgotPasswordError, onSubmit }) =>
     <Grid container spacing={24} className={classes.forgotPwdMainContainer}>
       <Grid item xs={12} sm={12} md={12} lg={12} className={classes.forgotPwdContent}>
         <Typography variant="h3">{title}</Typography>
```
Comment (raamb) [inline packages/shared/src/components/SNETForgotPassword/index.js:39]: typo?

File: packages/shared/dist/components/SNETForgotPasswordConfirm/index.js (hunk 2 above)
Comment (Vivek205) [reply packages/shared/dist/components/SNETForgotPasswordConfirm/index.js:42]: yh. this is auto generated file... Dist folder could be ignored

File: packages/shared/dist/components/SNETHeader/MobileHeader/index.js (hunk 3 above)
Comment (Vivek205) [reply packages/shared/dist/components/SNETHeader/MobileHeader/index.js:31]: yh. this is auto generated file... Dist folder could be ignored

File: packages/shared/src/components/SNETForgotPassword/index.js (hunk 4 above)
Comment (Vivek205) [reply packages/shared/src/components/SNETForgotPassword/index.js:39]: typography is a MaterialUI component

**Pull request merged**